components/availability/src/Availability.svelte

```diff
@@ -47,23 +45,39 @@
 
   // map over the ticks() of the time scale between your start day and end day
   // populate them with as many slots as your start_hour, end_hour, and slot_size dictate
+
+  $: generateDaySlots = function (timestamp, start_hour, end_hour) {
+    const dayStart = d3.timeHour(new Date(timestamp).setHours(start_hour));
+    const dayEnd = d3.timeHour(new Date(timestamp).setHours(end_hour));
```

Comment:
Are `start_hour` and `end_hour` based on a 24 hour or 12 hour clock? unix timestamp?

Reply:
Great question. Integer, 0-23. Done this way so users can pass in how much of the visible day to show without having to know specific dates or timestamps.